src/ComplexTypes/GetSignatureRequest.php

```diff
@@ -14,23 +14,23 @@ class GetSignatureRequest extends BaseType
     protected $Customer = null;
 
     /**
-     * @var RequestSignature $Shipment
+     * @var RequestShipment $Shipment
```

Comment:
I'm actually not sure about this one.

I was just looking at the current version in the repo:
https://github.com/danslo/Postnl/blob/54feb5a430ae4b30ba2de2b6d755654ead4c296a/src/ComplexTypes/GetSignatureRequest.php

Notice it was calling setSignature. But that method doesn't event exist. setShipment does exist so I (maybe wrongly?) assumed that it should be a RequestShipment instead.

I'll remove the change from my diff, but I think this is still a bug in the constructor at the very least.